features/foutafhandeling.feature

```diff
@@ -0,0 +1,542 @@
+        | ?locatie=98095.02,438495.09&bouwjaar[min]=1950&bouwjaar[max]=1979                 |
+
+Rule: de oppervlakte van de bounding box mag maximaal 250 duizend vierkante meter zijn.
+
+    Scenario: waarde van bbox parameter overschrijdt maximale toegestane oppervlakte 
+        Als '/panden?bbox=134647,457842,137512,455907' wordt aangeroepen
+        Dan bevat de response ten minste de volgende velden
+        | naam   | waarde                                        |
+        | title  | Een of meerdere parameters zijn niet correct. |
+        | status | 400                                           |
+        | code   | paramsValidation                              |
+        En bevat de response de volgende invalidParams
+        | name | code           | reason                                             |
+        | bbox | surfaceMaximum | Waarde is hoger dan maximum oppervlakte 250000 m2. |
+
+Rule: gebruik van de parameter oppervlakte is niet toegestaan in combinatie met type=standplaats of type=ligplaats
+
+    Abstract Scenario: zoeken met oppervlakte van standplaats of ligplaats
+        Als '/adresseerbareobjecten?bbox=135207,457400,135418,457162&oppervlakte[max]=100&type=<type>' wordt aangeroepen
+        Dan bevat de response ten minste de volgende velden
+        | naam   | waarde                                                                              |
+        | title  | De combinatie van opgegeven parameters is niet toegestaan.                          |
+        | status | 400                                                                                 |
+        | code   | unsupportedCombi                                                                    |
+        En bevat de response geen veld "invalidParams"
+
+        Voorbeelden:
+        | type        |
+        | standplaats |
+        | ligplaats   |      
+
+Rule: min waarde mag niet groter zijn dan max waarde
+
+    Abstract Scenario: opgegeven minimale waarde is groter dan opgegeven maximale waarde
+        Als '<path><query string>' wordt aangeroepen
+        Dan bevat de response ten minste de volgende velden
+        | naam   | waarde                                        |
+        | title  | Een of meerdere parameters zijn niet correct. |
+        | status | 400                                           |
+        | code   | paramsValidation                              |
+        En bevat de response de volgende invalidParams
+        | name             | code   | reason                |
+        | <parameter>[min] | <code> | <reason omschrijving> |
+
+        Voorbeelden:
+        | path                   | query string                                                                | parameter   | code  | reason omschrijving              |
+        | /adresseerbareobjecten | ?bbox=135207,457400,135418,457162&oppervlakte[min]=200&oppervlakte[max]=100 | oppervlakte | range | min mag niet hoger zijn dan max. |
```

Comment:
/adresseerbareobjecten?bbox=135207,457400,135418,457162&oppervlakte[min]=200&oppervlakte[max]=100 geeft:
```
{
    "type": "https://tools.ietf.org/html/rfc7231#section-6.5.1",
    "title": "Een of meerdere parameters zijn niet correct.",
    "status": 400,
    "detail": "Bad request.",
    "instance": "https://api.bag.acceptatie.kadaster.nl/esd/huidigebevragingen/v1/adresseerbareobjecten?bbox=135207,457400,135418,457162&oppervlakte[min]=200&oppervlakte[max]=100",
    "code": "paramsValidation",
    "invalidParams": [
        {
            "name": "oppervlakte",
            "code": "range",
            "reason": "min mag niet hoger zijn dan max."
        }
    ]
}
```

/panden?bouwjaar[min]=2000&bouwjaar[max]=1000 geeft:
```
{
    "type": "https://tools.ietf.org/html/rfc7231#section-6.5.1",
    "title": "Ten minste één parameter moet worden opgegeven.",
    "status": 400,
    "detail": "Precies 1 parameter van adresseerbaarObjectIdentificatie, nummeraanduidingIdentificatie, locatie of bbox moet worden opgegeven",
    "instance": "https://api.bag.acceptatie.kadaster.nl/esd/huidigebevragingen/v1/panden?bouwjaar[min]=2000&bouwjaar[max]=1000",
    "code": "paramsRequired"
}
```

Reply:
in de tweede ontbrak de bbox